### ISSUE TITLE
"new scenario" not reseting ok the variables?

### ISSUE THREAD

**Issue by diego2013** [diego2013/CSR_Meteor#68]
Workflow was the following:
- Submitted a scenario. Last screen visited was the "references" tab from step 2. Submited for approval
- Submitted feedback (via the button on the nav bar)
- Clicked the button "Enter new Scenario" on the "FeedbackFormThankyou panel. 

It displayed the "setp1" button in red, but also displayed the "references tab" from step two. 

Maybe the session variables to indicate the state/step where not reseted OK? The button leads to a route that is "/new" as opposed to the "newScenarioForm" from the navigation bar. Maybe the routing on "/new" is not doing something that "/newScenarioForm" does.

**Comment by diego2013** [comment]
![image](https://cloud.githubusercontent.com/assets/4389491/8658934/1c4155d6-2975-11e5-8679-f861401940f6.png)

Image above is what you get following the described workflow.

The explanation from previous comments makes sense, since it is doing 

```
this.route('new', function(){
  this.render('NewScenarioForm');
});
```

and does not have in consideration the other things the other routes (such as _newScenarioForm_ are doing).

**Comment by diego2013** [comment]
Fixed on v0.0.14
